framework/src/mesh/FaceInfo.C

```diff
@@ -93,3 +103,15 @@ FaceInfo::skewnessCorrectionVector() const
 
   return _face_centroid - r_intersection;
 }
+
+bool
+FaceInfo::varDefinedOnElem(const std::string & var_name) const
+{
+  auto it = _face_types_by_var.find(var_name);
```

Comment:
with this indexing by name, calling variables vel_x vel_y vel_z will be (likely infinitesimally) slower than u, v, w

Reply:
I guess we ll have to check profiles to see if this lookup done everywhere matters

Reply:
I like `u,v,w` and if we can include them in a vector variable for paraview I am open to changing them back. Also I am not a huge fan of this indexing. This might change when we introduce the consumer-dependent `isBoundary()` function to the variable and other functors.

Reply:
Hmm I should have commented on this earlier. I just searched through the diff (without expanding every file admittedly), and the usage of this I found in `uncorrectedAdGradSln` could be replaced by `this->hasBlocks(&fi.elem())` which would remove the string comparisons. Does this new method give us something that we didn't have before in other locations of the diff?

Reply:
No, these are the only locations where this routine is used. I was not aware of this `hasBlocks` approach, I'll change this and get rid of this method.